src/components/top-bar/_top-bar.scss

```diff
@@ -12,6 +12,7 @@
   width: 100%;
   top: 0;
   left: 0;
+  z-index: 1;
```

Comment:
why the changes was applied in previous PR?

Reply:
According to the last (current) implementation made at the end it actually wasn't needed